hack/machineset.sh

```diff
@@ -154,10 +154,7 @@ get_azure_ms() {
   local sku="2022-datacenter-smalldisk"
   local release="latest"
   if [ "$winver" == "2019" ]; then
-		# 2019 images without the containers feature pre-installed cannot be used due to
-		# https://issues.redhat.com/browse/OCPBUGS-13244
-    sku="2019-datacenter-with-containers-smalldisk"
-    release="17763.6293.240905"   
```

Comment:
are you sure the specific release can be removed?
Is the OpenSSH issue resolved?